From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: John Kleinschmidt <jkleinsc@electronjs.org>
Date: Tue, 1 Jul 2025 08:18:41 -0400
Subject: build: don't use large for clang on Windows

Chromium uses machines labeled as "large" in their
RBE cluster, but all of Electron's machines are
large so we don't specify it.

diff --git a/build/config/siso/clang_windows.star b/build/config/siso/clang_windows.star
index f1deb0e0664d5abec1d17f3a6d3942a744c4e3e8..604fa50d7e54baa037146fcbd9996a345f67f351 100644
--- a/build/config/siso/clang_windows.star
+++ b/build/config/siso/clang_windows.star
@@ -42,7 +42,6 @@ def __step_config(ctx, step_config):
         # no "action_large" Windows worker pool
         use_windows_worker = True
         if reproxy_config["platform"]["OSFamily"] != "Windows":
-            largePlatform["label:action_large"] = "1"
             use_windows_worker = False
         step_config["platforms"].update({
             "clang-cl": reproxy_config["platform"],
